bridge-fungible-token/tests/utils/builder.rs

```diff
@@ -22,10 +22,11 @@ pub async fn get_contract_message_script() -> (Vec<u8>, Bytes32) {
 }
 
 /// Gets the message to contract predicate
-pub async fn get_contract_message_predicate() -> (Vec<u8>, Address) {
+pub async fn get_contract_message_predicate() -> (Vec<u8>, Address, [u8; 32]) {
```

Comment:
the last return `[u8; 32]` is never used

Reply:
Ah, good catch! I was formerly encoding this in the message data until I realized it wasn't actually needed.